prime-router/metadata/fhir_mapping/hl7/secondary/PatientContact.yml

```diff
@@ -53,6 +53,17 @@ telecom:
   vars:
     valueIn: NK1.5
 
+telecom_business:
```

Comment:
Is there already just a telecom, curious why it is telecom_business instead of just telecom when the FHIR attribute column just says telecom

Reply:
The names put in these configs are just for our usage, and are made unique to make it easy to search the codebase. We have some anti-duplicate logic for them so if we have multiple `telecom` fields, it will throw an error